KsApi/mutations/ConfirmBackingAddressMutationTests.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+@testable import KsApi
+import XCTest
+
+final class ConfirmBackingAddressMutationTests: XCTestCase {
```

Comment:
These tests are a little silly since they're just exercising the `MockService` 🤷‍♀️. But at least they provide a little documentation of what the Service call can do.